src/index.js

```diff
@@ -0,0 +1,56 @@
+// import _ from 'lodash';
+// eslint-disable-line
+// import './style.css';
```

Comment:
- Please remove all commented code from your file. This would help make your code look more readable and easier to maintain 👍 👍